docs/changelog.rst

```diff
@@ -16,6 +16,11 @@ Core API
 - Start ``tlid`` counting at 1 instead of 0 to keep in sync with MPD's
   ``songid``.
 
+- `get_time_position` now returns the seek target while a seek is in progress.
+  This gives better results than just failing the position query.
+
+  (Fixes: :issue:`312` PR: :pr:`1346`)
```

Comment:
Put this in the same paragraph as the rest.

We don't have support for `:pr:` yet, it was only suggested on IRC. Feel free to add it to `docs/conf.py`.